webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -38,6 +39,14 @@ const AnsibleHostDetail = ({ response, status }) => {
               <AnsibleVariableOverrides id={response.id} />
             </TabLayout>
           </Tab>
+          <Tab
+            eventKey="jobs"
+            title={<TabTitleText>{__('Jobs')}</TabTitleText>}
```

Comment:
Should this be `Configuration runs` or something? Jobs sounds very generic and missleading

Reply:
It is in mockups. Any thoughts on that @MariSvirik?